packages/xo-web/src/xo-app/user/index.js

```diff
@@ -400,6 +401,18 @@ export default class User extends Component {
             </Col>
           </Row>
           <br />
+          <Row>
+            <Col smallSize={10} offset={2}>
+              <ActionButton
```

Comment:
What do you think of adding a modal when user change the password instead of Button?

Reply:
I also want to let the user sign out other sessions without having to change his password (e.g. he forgot to sign out from another computer but he knows that no-one knows his password).

So there are 2 different features:
- signing out all other sessions as a standalone action 
- automatically signing out other sessions after changing the password (I don't see any big benefit of making it optional, here)